box/jobs/fetch_upcoming_statements.rb

```diff
@@ -38,6 +38,7 @@ def fetch_for_account(account)
 
         vmk_data = account.transport_client.VMK(safe_from.to_s(:db), safe_to.to_s(:db))
         return unless vmk_data
+        vmk_data = vmk_data.encode(vmk_data.encoding, universal_newline: true)
```

Comment:
This seems kinda weird updating the variable itself instead of having a decoded_vmk_data or something similar. How did this work before? Does it not need to be encoded?

Reply:
Ah because vmk_data is just a string? Maybe just something like
```suggestion
     vmk_data = account
       .transport_client.VMK(safe_from.to_s(:db), safe_to.to_s(:db))
       &.encode(vmk_data.encoding, universal_newline: true) # converting windows newlines
        return unless vmk_data
```

Reply:
The encoding is handled in the gem itself